src/components/TermsAndPrivacy/Privacy.tsx

```diff
@@ -0,0 +1,119 @@
+import { Box, Flex, Heading, Link, List, ListItem, Text } from '@chakra-ui/react'
+
+const Privacy = () => {
+  return (
```

Comment:
Same as in terms. No need to use `{ return` when there's no logic in the component.